.travis/before_install_test.sh

```diff
@@ -12,3 +12,9 @@ sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 if [ "$SYMFONY" != "" ]; then composer require "symfony/symfony:$SYMFONY" --no-update; fi;
 if [ "$SONATA_CORE" != "" ]; then composer require "sonata-project/core-bundle:$SONATA_CORE" --no-update; fi;
 if [ "$SONATA_BLOCK" != "" ]; then composer require "sonata-project/block-bundle:$SONATA_BLOCK" --no-update; fi;
+if [ "${TRAVIS_PHP_VERSION:0:3}" != "5.6" ]; then
+    # but only with Symfony 3.4 and higher
+    if [[ -z "${SYMFONY}" || ("${SYMFONY:0:3}" != "2.8" && "${SYMFONY:0:3}" != "3.3") ]]; then
+        composer require "{{ "symfony/maker-bundle:${" ~ package_name|upper ~ ":=" ~ package_versions|first ~ "}"}}" --no-update
```

Comment:
I guess the twig variables are undefined/unknown

Reply:
LOL, I forgot to change that, nice catch :smile:

Reply:
:)